apps/chat/src/store/application/application.epics.ts

```diff
@@ -312,19 +312,16 @@ const updateApplicationEpic: AppEpic = (action$, state$) =>
                     pathname: payload.redirectUrl,
                     query,
                   });
-                } else if (state$.value.application.exitAfterSave) {
-                  Router.push({
-                    pathname: Routes.Marketplace,
-                    query: { tab: MarketplaceTabs.MY_WORKSPACE },
-                  });
                 }
 
-                return of(
-                  ApplicationActions.updateSuccess(updatedCustomApplication),
-                  ModelsActions.updateModel({
-                    model: updatedCustomApplication,
-                    oldApplicationId: payload.oldApplication.id,
-                  }),
+                return concat(
+                  of(
+                    ApplicationActions.updateSuccess(updatedCustomApplication),
+                    ModelsActions.updateModel({
+                      model: updatedCustomApplication,
+                      oldApplicationId: payload.oldApplication.id,
+                    }),
+                  ),
```

Comment:
```suggestion
                return of(
                  ApplicationActions.updateSuccess(updatedCustomApplication),
                  ModelsActions.updateModel({
                    model: updatedCustomApplication,
                    oldApplicationId: payload.oldApplication.id,
                  }),
```

do we actually need to wrap this with concat?

Reply:
You are wright it's redundant.